docker-compose.development.yml

```diff
@@ -15,6 +16,8 @@ services:
       POSTGRES_PORT: 5432
       POSTGRES_SCHEMA: public
     command: poetry run python scripts/start_api.py --host 0.0.0.0
+    volumes:
+      - ./data:/data
```

Comment:
Does OQT every write to this directory? If yes, this could lead to issues.

Reply:
No. That is not planed. It should only read. 
As of now, only the `rasterstats` package accesses those raster. `rasterstats` only reads those files.